helm/ingress-controller/tests/__snapshot__/controller-rbac_test.yaml.snap

```diff
@@ -0,0 +1,103 @@
+Should generate rbac:
```

Comment:
nit ^ the `.keep` file doesn't seem necessary anymore

Reply:
Fixed in https://github.com/ngrok/ngrok-ingress-controller/pull/69/commits/ae6973830a6227706075691ce69609a8b700ea5b